.env.sample

```diff
@@ -7,3 +7,5 @@ DATABASE_SSL=disable
 DATABASE_USER=postgres
 ENVIRONMENT=development
 TZ=Europe/Berlin
+TIBIA_WORLD_NAME=tibia_world_name
+TIBIA_WORLD_API_BASE_URL=world_api_url:port
```

Comment:
Let's include actual format example, so it's clear if we expect `http(s)://`, `tcp://` or no protocol descriptor. Also it should specify if there is an API version included.

e.g.

`TIBIA_WORLD_API_BASE_URL=https://tibiadata.com/api/v2`